src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -100,36 +101,32 @@ private void ItemManipulationModel_ScrollIntoViewInvoked(object? sender, ListedI
 		}
 
 		private void ItemManipulationModel_StartRenameItemInvoked(object? sender, EventArgs e)
-		{
-			StartRenameItem();
-		}
+			=> StartRenameItem();
```

Comment:
I think methods should have curly braces, @d2dyno1

Reply:
Yep, could you revert it @ferrariofilippo?

We should update codestyle guidelines because I've noticed @cinqmilleans doing it too

Reply:
> Yep, could you revert it @ferrariofilippo?
No problem, I'll revert it in my other PRs too

Reply:
Thank you!

Reply:
Done